### DISCUSSION TITLE
bug report

### DISCUSSION THREAD

**Discussion by kw600** [SSCHAcode/python-sscha#146] [General]
I am wondering if there is any place I can report bugs. I just put it here now.

There is a bug in the save_qe() of Phonons.py. (line 1248). The original code is as follows:

                for i in range(n_atoms):
                    fp.write("( %10.6f%10.6f %10.6f%10.6f %10.6f%10.6f )\n" %
                             (np.real(atomic_disp[3*i, mu]), np.imag(atomic_disp[3*i,mu]),
                              np.real(atomic_disp[3*i+1, mu]), np.imag(atomic_disp[3*i+1,mu]),
                              np.real(atomic_disp[3*i+2, mu]), np.imag(atomic_disp[3*i+1,mu])))

Look at the last line np.real(atomic_disp[3*i+2, mu]), np.imag(atomic_disp[3*i+1,mu])
The second imaginary part index should be [3*i+2,mu] instead of [3*i+1,mu].
This makes the imaginary part disapper in the calculations of out of plane vibrations.

**Comment by mesonepigreco** [comment]
Hi thanks a lot for this bug report. 
Usually, this should go in Issues (inside the cellconstructor repository), but I will fix it right away.

**Reply by mesonepigreco** [reply]
https://github.com/SSCHAcode/CellConstructor/issues/76

Fixed on the main repo of CellConstructor.
The release 1.4 (soon to come) will contain the fix
